spark-submit-flamegraph

```diff
@@ -237,7 +237,9 @@ main() {
   influx_uri=http://${local_ip}:${influx_http_port}
   flamegraph_title="Spark Application"
 
-  if [ -n "${ONLY_INSTALL_DEPS}" ]; then
+  if [ -z "${ONLY_INSTALL_DEPS+proceed}" ]; then
+    echo "Proceeding..."
```

Comment:
Why do we need this message?

Reply:
I guess we don't need it. Let me check...